resource/in

```diff
@@ -20,27 +21,36 @@ def get_client(dest_dir, source={}, version=None, params={}):
 
     auth0.access(ManagementAPI(source['domain']))
 
-    print(f'Fetching client {source["app-name"]} details ...')
+    log.msg("Fetching client.", name=source["app-name"])
```

Comment:
This is what I meant the other day, you'll have to play with it in a test pipeline but these messages like "Fetching client details..." are also useful to provide feedback to the user monitoring the pipeline regarding the progress.

I don't know if in this case wrap them in a JSON blob (which is machine readable but not-so human readable) is the best course of action.

But as I said, have a play ☺️

Reply:
I don't mind either way.

Don't know if maybe we could keep the timestamp but lose the JSON formatting for these non-exception messages? Just a thought, don't know what you reckon.

Reply:
Yeah... I did wonder this too while putting together this PR. I wonder if I could ensure the descriptive message ("Fetching client." etc...) was always the first item in the JSON object so folks could still at least see what's going on. I'll take a look.